test/main.cpp

```diff
@@ -0,0 +1,140 @@
+#include "University.hpp"
+#include "../argparse/argparse.h"
+
+using namespace argparse;
+
+std::vector<std::string> splitStringBySpace(const std::string& str)
+{
```

Comment:
Replace brackets to end of line
std::vector<std::string> splitStringBySpace(const std::string& str) {
(in all places)